### DISCUSSION TITLE
shortestPath() from Cypher in AGE

### DISCUSSION THREAD

**Discussion by tkluza** [apache/age#248] [General]
Hi everyone!

That is my first thread on this project. I am currently in research for finding proper solution for our needs and I came across this very promising project. 

I am curious whether is it possible to use `shortestPath()` method, which is accessible via Cypher? I don't see that in documentation. Is it possible? When not, is it planned for future release?

Best regards!

**Comment by jrgemignani** [comment]
@tkluza shortestPath() is planned for future releases. However, we don't have any definitive schedule for it at the moment.

**Reply by tkluza** [reply]
@jrgemignani Thank you for a such quick response. I am going to watch for future releases then.
